docs/source/tutorials/base/how_to_train/classification.rst

```diff
@@ -1,7 +1,234 @@
+  | CLASSIFICATION |   Custom_Image_Classification_EfficientNet-V2-S   |   EfficientNet-V2-S   |   otx/algorithms/classification/configs/efficientnet_v2_s_cls_incr/template.yaml  |
+  +----------------+---------------------------------------------------+-----------------------+-----------------------------------------------------------------------------------+
+
+To have a specific example in this tutorial, all commands will be run on the :ref:`MobileNet-V3-large-1x <classificaiton_models>`  model. It's a light model, that achieves competitive accuracy while keeping the inference fast.
+
+2.  Next, we need to create train/validation sets. OTX supports auto-split functionality for the multi-class classificaiton. For other classification types we need to prepare splits in advance
+Let's prepare an OTX classification workspase running the following command:
+
+.. note::
+
+  Currently, OTX supports auto-split only for multi-class classificaiton. For the multi-label and hierarchical tasks we need to prepare data splits in advance.
+
+.. code-block::
+
+  (otx) ...$ otx build --train-data-roots data/flower_photos --model MobileNet-V3-large-1x
+
+  [*] Load Model Template ID: Custom_Image_Classification_MobileNet-V3-large-1x
+  [*] Load Model Name: MobileNet-V3-large-1x
+  [*] Saving data configuration file to: ./otx-workspace-CLASSIFICATION-MobileNet-V3-large-1x/data.yaml
+
+  (otx) ...$ cd ./otx-workspace-CLASSIFICATION-MobileNet-V3-large-1x
+
+It will create **otx-workspace-CLASSIFICATION** with all necessery configs for MobileNet-V3-large-1x, prepared ``data.yaml`` to simplify CLI commands launch and splitted dataset
+
+2. To start training we need to call ``otx train`` command in our worspace:
+
+.. code-block::
+
+  (otx) ...$ otx train
+
+That's it! The training will return artifacts: ``weights.pth`` and ``label_schema.json``, which are needed as input for the further commands: ``export``, ``eval``,  ``optimize``,  etc.
+
+The training time highly relies on the hardware characteristics, for example on 1 GeForce 3090 the training took about 8 minutes.
+
+After that, we have the PyTorch classification model trained with OTX, which we can use for evaluation, export, optimization and deployment.
+
+***********
+Validation
+***********
+
+1. ``otx eval`` runs evaluation of a trained
+model on a specific dataset.
+
+The eval function receives test annotation information and model snapshot, trained in the previous step.
+Please note, ``label_schema.json`` file contains meta-information about the dataset and it should be located in the same folder as the model snapshot.
+
+otx eval will output a top-1 accuracy score for multi-class classification
+
+2. The command below will run validation on our dataset
+and save performance results in ``performance.json`` file:
+
+.. code-block::
+
+  (otx) ...$ otx eval --test-data-roots splitted_dataset/val \
+                      --load-weights results/weights.pth \
```

Comment:
If only enter the `otx train` command (without `--save-model-to`), now save it to `models/weights.pth` (default).
Also, if do not input `--load-weight` in otx eval (including other commands), it will try to find `models/weights.pth` as the default value. Raises an error if it is not present in both args `--load-weight` and default path.

'+

If there is no `--save-performanc-to`, performance.json is created in the folder where the model weight is located by default.

For each command, there is a case where it is changed to an option and a default value is given. In fact, in the documentation tutorial, it would be okay to induce user to give command args.

Reply:
So I'd like to say here that only the folder name `results` is wrong (-> `models`) with current implementation.

Reply:
Thank you for your kind explanation. I just followed Sungman PR that used "results" instead of models. But now this PR is closed...